docs/designers-developers/developers/block-api/block-registration.md

```diff
@@ -716,6 +716,14 @@ attributes: {
 alignWide: false,
 ```
 
+-   `defaultStyle` (default `true`): When the style picker is shown, a dropdown is displayed so the user can select a default style for this block type. If you prefer not to show the dropdown, set this property to `false`.
```

Comment:
Noting that the editor setting itself is still marked as experimental:

https://github.com/WordPress/gutenberg/blob/8a94254b6adac030f5a6ffd8e718972e1f19f48e/packages/block-editor/src/components/default-style-picker/index.js#L23

Which leaves this in an awkward mix of semi-stable support.

Related: #16465

@jorgefilipecosta Do you have a sense whether this is something that will plan to be stabilized? Or at least that even if the setting shape itself changes, the desire for "default style" user control will remain?

Reply:
@jorgefilipecosta: Pinging you on this question: is there any plan to stabilise this feature?

Reply:
We still have a considerable amount of work to do on https://github.com/WordPress/gutenberg/issues/7551. Allowing the user to change the default styles was just a step.
We also want to allow themes to change default styles https://github.com/WordPress/gutenberg/pull/16532.
And we want to allow users to create their block styles from the UI. I think this API should not be stabilized before at least these two follow-ups are merged and tested (as the follow-up work may benefit from a better mechanism to store and pass default styles around the several artifacts we have). So there are no immediate plans for stabilization.